crates/common/chain/lean/src/lean_chain.rs

```diff
@@ -401,7 +400,7 @@ impl LeanChain {
     /// <https://github.com/leanEthereum/leanSpec/blob/ee16b19825a1f358b00a6fc2d7847be549daa03b/docs/client/forkchoice.md?plain=1#L279-L312>
     pub async fn on_attestation_from_block(
         &mut self,
-        signed_votes: impl IntoIterator<Item = SignedVote>,
+        signed_votes: impl IntoIterator<Item = SignedAttestation>,
```

Comment:
```suggestion
        signed_attestations: impl IntoIterator<Item = SignedAttestation>,
```

We're gonna drop all `vote` jargons, so I'd suggest to fix the parameter name as well.

Reply:
https://github.com/leanEthereum/leanSpec/pull/78